homeronet/Plugin/Weather.cs

```diff
@@ -0,0 +1,395 @@
+﻿using ForecastIO;
+using GeocodeSharp.Google;
+using homeronet.Client;
+using homeronet.Messages;
+using Ninject;
+using Ninject.Parameters;
+using SkiaSharp;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Drawing;
+using System.IO;
+using System;
+
+namespace homeronet.Plugin {
+
+    public class WeatherRendererInfo {
+        public ForecastIOResponse weather;
+        public string address;
+        public Unit unit;
+    }
+
+    public class WeatherRenderer {
+        SKTypeface tf = SKTypeface.FromFile("Resources/Weather/Star4000.ttf");
+        SKTypeface tfsm = SKTypeface.FromFile("Resources/Weather/Star4000 Small.ttf");
+        SKTypeface tflg = SKTypeface.FromFile("Resources/Weather/Star4000 Large.ttf");
+
+        SKColor colorYellow = new SKColor(205, 185, 0);
+        SKColor colorOrange = new SKColor(194, 108, 2);
+        SKColor colorDkBlue = new SKColor(41, 25, 92);
+        SKColor colorBlue = new SKColor(8, 15, 255);
+        SKColor colorLtBlue = new SKColor(121, 112, 255);
+        SKColor colorTeal = new SKColor(47, 65, 120);
+        SKColor colorTealAlso = new SkiaSharp.SKColor(172, 177, 237);
+        SKColor colorRed = new SKColor(198, 19, 2);
+
+        private Dictionary<string, string[]> forecastDescriptions = new Dictionary<string, string[]>() {
```

Comment:
Really don't like this. Why not replace all references to this dict by doing something like:

```
CultureInfo.InvariantCulture.TextInfo.ToTitleCase("clear-day".Replace('-',' '));
```